doc/developer-guides/hld/static-core-hld.rst

```diff
@@ -0,0 +1,138 @@
+.. cpu-partitioning-hld:
+
+Static CPU Partitioning
+#######################
+
+CPU partitioning is a policy for mapping a virtual
+CPU (VCPU) to a physical CPU. The current ACRN implementation forces a
+static 1:1 mapping between VCPUs and physical CPUs and does
+not support multiple VCPUs running on a physical CPU and does not
+support VCPU migration from one physical CPU to another.
+
+This section describes the static CPU partitioning implementation in
+ACRN.
+
+Overview
+********
+
+ACRN forces a fixed 1:1 mapping between a VCPU and a physical CPU when
+creating a VCPU for the guest Operating System. This makes the VCPU
+management code much simpler.
+
+An array is used to track the physical CPU allocation information. When
+a VCPU is created, we query, allocate, and update the array to map the
+VCPU to an available physical CPU.
+
+The physical CPU number assignment for each guest is pre-defined. For
+example, on a platform with four CPU cores, one physical CPU is assigned
+to run the Service Operating System (SOS) and other three physical CPUs
+are assigned to run the User Operating System (UOS) instances.
+
+.. note::
+
+   To improvement SOS boot time, all physical CPUs are assigned to the SOS
+   during the SOS boot. Afterward, the physical CPUs defined for the UOS
+   are allocated by the Device Model (DM) by running the launch_uos.sh
+   script.
+
+CPU management in SOS
+*********************
+
+With ACRN, all ACPI table entries are pass-thru to the SOS, including
+the Multiple Interrupt Controller Table (MADT). The SOS sees all
```

Comment:
Per https://wiki.osdev.org/MADT, "MADT" stands for Multiple APIC Description Table, please confirm the acronym of "Multiple Interrupt Controller Table".